### DISCUSSION TITLE
Error Visual Studio Code

### DISCUSSION THREAD

**Discussion by Xarlight** [microsoft/vscode-cpptools#12141] [Q&A]
Hello everyone,

I am currently a second year student and following a project I find myself using Visual Studio Code however after many tries a perpetual error to launch my code remains (I tried some solution on the net but nothing works). I was wondering if you had a solution, so I'll take the liberty of posting a few photos:

My code :

![image](https://github.com/microsoft/vscode-cpptools/assets/102165890/57d079c3-81b2-4780-ae0f-a8c9c8dacfdc)

My c_ cpp_proprietes.json :

![image](https://github.com/microsoft/vscode-cpptools/assets/102165890/cbc5be51-fcc1-46c3-9e7e-938f2222fda6)

and I don't know if it's any use but my launch.json :

![image](https://github.com/microsoft/vscode-cpptools/assets/102165890/25d06014-6d25-408b-8f13-58905a193c47)

Thank you very much for taking the time to answer my question.

Have a nice day. :1st_place_medal:

**Comment by sean-mcmanus** [comment]
@Xarlight That is caused by clang-tidy and using gcc headers. The gcc headers are attempting to use __float128 when your clang compiler thinks it doesn't support them. I'm not sure if it's bug we can fix or not. We could convert this to an issue to potentially investigate later.
